docker/policies/run_using_sudo_test.rego

```diff
@@ -0,0 +1,107 @@
+package appshield.DS010
+
+test_basic_denied {
```

Comment:
We should have two denied cases since the policy split a value by `&&`.

1. basic

```
RUN sudo pip install --upgrade pip
```

2. `&&`

```
RUN apk add bash && sudo pip install --upgrade pip
```

Reply:
done